packages/faucet/src/faucet.ts

```diff
@@ -50,10 +57,17 @@ export class Faucet {
     this.holderAddress = wallets[0][0];
     this.distributorAddresses = wallets.slice(1).map((pair) => pair[0]);
 
+    const fees: Partial<FeeTable> = {
+      send: {
+        amount: constants.fee,
+        gas: constants.gas,
+      },
+    };
+
     // we need one client per sender
     const clients: { [senderAddress: string]: SigningCosmosClient } = {};
     for (const [senderAddress, wallet] of wallets) {
-      clients[senderAddress] = new SigningCosmosClient(apiUrl, senderAddress, wallet);
+      clients[senderAddress] = new SigningCosmosClient(apiUrl, senderAddress, wallet, fees);
```

Comment:
Ahh... that was tricky, the cosm comes from the defaults in CosmosClient. Good find.